### ISSUE TITLE
Auto launch player should ignore %f and %U

### ISSUE THREAD

**Issue by GoogleCodeExporter** [memeco/osd-lyrics#232]
```
What steps will reproduce the problem?
1. Choose Audacious2 to launch automatically, remember it.
2. Start osd lyrics without any supported player running.

What is the expected output? What do you see instead?
Audacious is launched with an invalid track named %U. The %U should be ignored.


Please use labels and text to provide additional information.

```

Original issue reported on code.google.com by `tigerso...@gmail.com` on 6 Aug 2011 at 8:42

**Comment by GoogleCodeExporter** [comment]
```
Patch to this issue
```

Original comment by `tigerso...@gmail.com` on 6 Aug 2011 at 9:09

Attachments:
- [patch-autolaunch.diff](https://storage.googleapis.com/google-code-attachments/osd-lyrics/issue-232/comment-1/patch-autolaunch.diff)

**Comment by GoogleCodeExporter** [comment]
```
totally fixed in 0.4.2
```

Original comment by `tigerso...@gmail.com` on 25 Nov 2011 at 2:30
- Changed state: **Fixed**
